.github/workflows/build_test.yml

```diff
@@ -12,6 +20,7 @@ env:
 
 jobs:
   smoke_test:
+    if: "github.repository == 'numpy/numpy' && !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
     runs-on: ubuntu-latest
```

Comment:
I think this line can be broken, see https://stackoverflow.com/a/3790497.

Reply:
I did this on purpose actually. It may be necessary to copy this to each job (it is in SciPy, it _may_ not be needed here because the other jobs depend on `smoke_test`), and then longer vertical blocks for what's basically boilerplate becomes annoying.

I'd prefer to leave it as is for now.